compiler/passes/convert-typed-uast.cpp

```diff
@@ -4938,13 +4938,23 @@ Expr* TConverter::convertParenlessCallOrNull(const AstNode* node, RV& rv) {
   }
 
   auto calledFn = findOrConvertFunction(rf);
-  auto ret = new CallExpr(calledFn);
+  Expr* ret = nullptr;
 
   if (sig->isMethod()) {
-    TC_UNIMPL("Handle parenless method call!");
-    return TC_PLACEHOLDER(this);
+    if (node->isIdentifier()) {
+      INT_ASSERT(cur.fnSymbol && cur.fnSymbol->isMethod());
+      ret = new CallExpr(calledFn, cur.fnSymbol->_this);
+    } else {
+      auto [recvAst, fieldName] = accessExpressionDetails(node);
+      types::QualifiedType qtRecv;
+      auto recv = recvAst ? convertExpr(recvAst, rv, &qtRecv) : nullptr;
+      ret = new CallExpr(calledFn, recv);
```

Comment:
Is `qtRecv` an out-parameter here? it's not used anywhere, so it doesn't seem you need to pass it